sdks/go/pkg/beam/core/runtime/xlangx/expand.go

```diff
@@ -140,3 +142,67 @@ func QueryExpansionService(ctx context.Context, p *HandlerParams) (*jobpb.Expans
 	}
 	return res, nil
 }
+
+func startAutomatedExpansionService(gradleTarget string) (func() error, string, error) {
```

Comment:
Nit: I think for documentation purposes, some named return values would be really nice here. Naming the return values `stopFunc` and `address` would make it very clear what they do without requiring a comment.

Reply:
That makes sense. Changed.